### ISSUE TITLE
Bug in line chart dialog

### ISSUE THREAD

**Issue by Patowhiz** [africanmathsinitiative/R-Instat#7789]
There is a bug in dialog  **Describe > Specific Tables/Graphs > Line/Smooth/Dumbell/Slope Plot** Line Plot Option.

The script produced by the dialog in the first load is different from the one produced when I reload the dialog. 

**First load Script**

```
# Code generated by the dialog, Line

dodoma <- data_book$get_data_frame(data_name="dodoma")
last_graph <- ggplot2::ggplot(data=dodoma, mapping=ggplot2::aes(x=year, y=rain, group=1)) + ggplot2::geom_line() + theme_grey()
data_book$add_graph(graph_name="last_graph", graph=last_graph, data_name="dodoma")
data_book$get_graphs(data_name="dodoma", graph_name="last_graph")
rm(list=c("last_graph", "dodoma"))
```

**Re-load Script**
```
# Code generated by the dialog, Line

dodoma <- data_book$get_data_frame(data_name="dodoma")
last_graph <- ggplot2::ggplot(data=dodoma, mapping=ggplot2::aes(x=year, y=rain)) + ggplot2::geom_line() + theme_grey()
data_book$add_graph(graph_name="last_graph", graph=last_graph, data_name="dodoma")
data_book$get_graphs(data_name="dodoma", graph_name="last_graph")
rm(list=c("last_graph", "dodoma"))
```
Notice `group=1` is missing in the second load.

**Comment by Patowhiz** [comment]
This can be fixed after PR #7782 is merged. To prevent conflicts.

**Comment by Patowhiz** [comment]
@N-thony this can be fixed by any team member without risk of conflict. Thanks
